petab/parameter_mapping.py

```diff
@@ -281,49 +298,54 @@ def _apply_dynamic_parameter_overrides(mapping: ParMappingDict,
     for overridee_id in condition_df.columns:
         if overridee_id == CONDITION_NAME:
             continue
-        if condition_df[overridee_id].dtype != 'O':
```

Comment:
why not needed any more?

Reply:
We always overwrite the existing parameter, even if there is a string. They are replaced below.